app-benchmarks/unigine-superposition/unigine-superposition-1.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop xdg-utils
+
+DESCRIPTION="Game like benchmark (CPU, GPU), for ~5 minutes"
+HOMEPAGE="https://benchmark.unigine.com/superposition"
+
+SRC_URI="https://assets.unigine.com/d/Unigine_Superposition-${PV}.run"
+S="${DISTDIR}"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="strip"
```

Comment:
Also `mirror` and `bindist`.

Reply:
Thanks, done d69a838ba

Why restrict `mirror`?

`bindist`: every compiled package must have it?